services/galley/src/Galley/API/Action.hs

```diff
@@ -691,22 +694,22 @@ addMembersToLocalConversation lcnv users role = do
 
 notifyConversationAction ::
   forall tag r.
-  ( Member FederatorAccess r,
+  ( Member (Error FederationError) r,
+    Member FederatorAccess r,
     Member ExternalAccess r,
     Member GundeckAccess r,
     Member (Input UTCTime) r,
     Member (Logger (Log.Msg -> Log.Msg)) r
   ) =>
-  Bool ->
   Sing tag ->
   Qualified UserId ->
   Bool ->
   Maybe ConnId ->
   Local Conversation ->
   BotsAndMembers ->
   ConversationAction (tag :: ConversationActionTag) ->
-  Sem r LocalConversationUpdate
-notifyConversationAction failEarly tag quid notifyOrigDomain con lconv targets action = do
+  Sem r (LocalConversationUpdate, FailedToProcess)
+notifyConversationAction tag quid notifyOrigDomain con lconv targets action = do
```

Comment:
Can we use something other than bool here?

Reply:
The remaining `Bool` argument wasn't introduced with this PR. While in general I would consider converting it into a simple sum type that is more descriptive, I'd ask you to not consider this to be a blocker and leave that for future work, as this PR has been going on for quite some time now.